src/commands/__tests__/log.test.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-env mocha */
+/* global expect, testContext */
+/* eslint-disable prefer-arrow-callback, no-unused-expressions */
+
+// import nock from 'nock'
+
+describe(testContext(__filename), function () {
+  describe('invoke', function () {
+    before(function () {
+      this.invoke = require('../log').invoke
+      this.notify = msg => {
+        this.notifications.push(msg)
+      }
+      this.argv = ['-r1', 'some1:25', 'some2:25', 'some3:25', 'some4:25']
+      this.lgJWT = 'not.a.real.token'
+      this.lgPlayer = {id: 'not.a.real.id'}
+    })
+    beforeEach(function () {
+      this.notifications = []
+    })
+
+    it('notifies with the usage message when requested', function () {
+      const {lgJWT, lgPlayer} = this
+      this.invoke(['-h'], this.notify, {lgJWT, lgPlayer})
+      expect(this.notifications[0]).to.match(/Usage:/)
+    })
+
+    it('notifies with a usage hint when not logging reflections', function () {
+      const {lgJWT, lgPlayer} = this
+      this.invoke([], this.notify, {lgJWT, lgPlayer})
+      expect(this.notifications[0]).to.match(/\-\-help/)
+    })
+
+    // TODO: enable this test once APIs are ready
```

Comment:
since the API's are nocked out here, seems like these tests should be enable-able now. Not so?